### PR TITLE
Rsvp form page

### PR DESCRIPTION
**Pull request by iden-a**

### CODE REVIEW

File: views/js/evently/src/RSVPForm.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          console.log(err);
+        }
+        // else {
+        //     console.log("Please submit the field")
+        //     setErrorMessage("Please Submit All Fields!")
+        // }
+      }
+
+    return (
+        <>
```
Comment (mclausen792) [inline views/js/evently/src/RSVPForm.jsx:40]: The indentation in this file isn't consistent and it makes it a little hard to read. Can you fix it? I believe it should be like this 
```
<>
            <div className="create-rsvp">
                <div className='rsvp-form-container'>
                    <form className='rsvp-form' onSubmit={handleRSVPSubmit}>
                        <h2 className='rsvp-title'> 
                        Register for "Event Name" 
                        </h2>

                        <div className="eventid-rsvp-container">
                            <div className='eventId'>
                                <label> Event ID: </label>
                                <input type="number" className = "rsvp-input-boxes" value={eventId} onChange={(e) => setEventId(e.target.value)} />
                            </div>

                            <div className='response-container'>
                                <label> RSVP: </label>
                                <select className = "rsvp-input-boxes" value={RSVPResponse} onChange={(e) => setRSVPResponse(e.target.value)}>
                                    <option name="yes" value="yes" > Yes </option>
                                    <option name="no" value="no" > No </option>
                                    <option name="maybe" value="maybe" > Maybe </option>
                                </select>
                            </div>
                        </div>

                        <div className='first-last-name-container'>
                            <div className='first-name'>
                                <label> First Name: </label>
                                <input type="text" className = "rsvp-input-boxes" value={firstName} onChange={(e) => setFirstName(e.target.value)} />
                            </div>

                            <div className='last-name'>
                                <label> Last Name: </label>
                                <input type="text" className = "rsvp-input-boxes" value={lastName} onChange={(e) => setLastName(e.target.value)} />
                            </div>
                        </div>

                        <div className="phone-email-container">
                            <div className='phone'>
                                <label> Phone #: </label>
                                <input type="text" className = "rsvp-input-boxes" value={phoneNumber} onChange={(e) => setPhoneNumber(e.target.value)} />
                            </div>

                            <div className='email'>
                                <label> Email: </label>
                                <input type="text" className = "rsvp-input-boxes" value={email} onChange={(e) => setEmail(e.target.value)} />
                            </div>
                        </div>


                        <button type='submit' className='rsvp-submit-btn'> Register</button>

                    </form>

                    {successMessage && (
                    <div role="alert" className="alert">
                        <div>{successMessage}</div>
                    </div>)}
                    {/* {errorMessage && (
                    <div role="alert" className="error-alert">
                    <div>{errorMessage}</div>
                    </div>)} */}
                </div>
            </div>
</>
```

**Review by mclausen792 — CHANGES_REQUESTED** [review]
Nice work! Just a couple little requests

File: views/js/evently/src/RSVPForm.jsx (hunk 1 above)
Comment (iden-a) [reply views/js/evently/src/RSVPForm.jsx:40]: Okay, I will update the file with these changes.

**Review by mclausen792 — APPROVED** [review]
Looks good, thanks for making those updates!

**Pull request merged**